tests/test.nf.test

```diff
@@ -53,7 +53,13 @@ nextflow_pipeline {
                         path("$outputDir/malt/malt_index/taxonomy.tre"),
                         "malt_index/table0.db - exists: ${path("$outputDir/malt/malt_index/table0.db").exists()}",
                         "malt_index/table0.idx - exists: ${path("$outputDir/malt/malt_index/table0.idx").exists()}",
-                        "downstream_samplesheets/databases-taxprofiler.csv - nr. lines: ${path("$outputDir/downstream_samplesheets/databases-taxprofiler.csv").readLines().size() == 8}",
+                        path("$outputDir/kmcp/database-kmcp-index/R001/_block001.uniki").exists(),
+                        path("$outputDir/kmcp/database-kmcp-index/R001/__db.yml").exists(),
+                        path("$outputDir/kmcp/database-kmcp-index/R001/__name_mapping.tsv").exists(),
```

Comment:
Are there any stable strings here we can rely on rather than just file existence, so we can ensure they actually include something and are not empty?

Reply:
Here, the first file `_block001.uniki` is a tool-specific file as I understand, see for instance [KMCP: Accurate metagenomic profiling](https://bioinf.shenwei.me/kmcp/#1-accurate-metagenomic-profiling), and is not readable as a text file. So I guess I could add a file size check instead. The other two can have a string check.